internal/store/files/README.md

```diff
@@ -0,0 +1,21 @@
+## BQ SQL query for WorldGeosForPlaceRecognitionAbbreviatedNames.csv
```

Comment:
We could also expand to `isoCode` for AA1's which has codes like "US-CA" from which we parse out the last two chars... so TN (for Tamil Nadu state), SH (for shanghai).  But we might prefer US states if we do that..

Reply:
We can have TODO for now, and expand later, per Guha suggestion.

Reply:
Added a TODO in the README.md which contains the BQ SQL query.